draft-ietf-tls-dtls-connection-id.md

```diff
@@ -345,6 +345,30 @@ data the following modification is made to the additional data calculation.
                       length_of_DTLSInnerPlaintext;
 ~~~
 
+# Peer Address Update Checks
+
+When a record with CID is received that has the source address of the enclosing
+UDP datagram different from the one previously recorded, the receiver MUST NOT
+update its view of the peer address with the source specified in the UDP packet
+before cryptographically validating the enclosed record(s).
+
+Immediately after reception, the recipient SHOULD initiate address validation
+using a Heartbeat {!RFC6520} based ping-pong with the new peer address.  This
+is to ensure that address validation can be cryptographically verified.  Both
+Heartbeat messages (request and response) MUST be wrapped into tls12_cid
+records.  If CID has not been negotiated, the sender MUST use a 0-length CID
+instead of the RFC 6347-defined record format. While the address validation
```

Comment:
My feeling is, tls12_cid with empty cid was discussed some months ago. To change this, would cause changes in other parts (MAC), and I'm not sure, if this pays off. Assuming, that the response will be a tls12_cid, which risk do you see, which cause the MUST for the request?